micrometer-core/src/test/java/io/micrometer/core/aop/TimedAspectTest.java

```diff
@@ -525,6 +546,11 @@ void getAnnotationForTagValueExpression(
             @Timed
             void getAnnotationForArgumentToString(@MeterTag("test") Long param);
 
+            @Timed
+            void getMultipleAnnotationForTagValueExpression(
```

Comment:
Do you think it would worth adding one more test case that uses `MeterTags` explicitly (instead of Java 8's repeatable annotations)?

Reply:
good point, added